Post/tests.py

```diff
@@ -93,3 +96,106 @@ def form():
             'Description': description,
         }
     )
+
+
+pytest.mark.django_db
+
+
+class CreatePostTest(TestCase):
```

Comment:
We are using django-pytest frame work, not unit test, please use client helper.
Review UI talk in class